index.js

```diff
@@ -0,0 +1,94 @@
+
+console.log("Hello");
+function playRound(playerSelection_, computerSelection_) {
+  // your code here!
+  playerSelection = playerSelection_.toLowerCase();
+  computerSelection = computerSelection_.toLowerCase();
+  if(playerSelection == "rock"){
+    if(computerSelection == "rock"){
+        console.log("tie");
+        return "None";
+    }
+    else if(computerSelection == "papper"){
+        console.log("Lose");
+        return "Lose";
+        //computer_count++;
+    }
+    else{
+        console.log("Win");
+        return "Win";
+        //player_count++;
+    }
+
+  }
+  else if(playerSelection == "papper"){
+   if(computerSelection == "papper"){
+    console.log("tie");
+    return "None";
+   }
+   else if(computerSelection == "rock"){
+    console.log("Win");
+    return "Win";
+    //player_count++;
+   }
+   else{
+    console.log("Lose");
+    return "Lose";
+    //computer_count++;
+   }
+  }
+  else{
+   if(computerSelection == "papper"){
+    console.log("Win");
+    return "Win";
+    //player_count++;
+   }
+   else if(computerSelection == "rock"){
+    console.log("Lose");
+    return "Lose";
+    //computer_count++;
+   }
+   else{
+    console.log("tie");
+    return "None";
+   }
+  }
+
+}
+let choices = ['Rock', 'Papper', 'Scissors'];
+function getComputerChoice(){
+    let result = choices[Math.floor(Math.random() * choices.length)];
+    return result;
+}
+let computerSelection = getComputerChoice();
+//console.log(playRound(playerSelection, computerSelection));
+
+function game() {
+  let computer_count = 0;
+let player_count = 0;
```

Comment:
JS variable naming convention does not follow snake_case. snake_case is common in python. In JS, we use camelCase. Pls update.